templates/single-tapestry.php

```diff
@@ -48,24 +48,6 @@
             var wpUserId = "<?php echo apply_filters('determine_current_user', false); ?>";
             var apiUrl = "<?php echo get_rest_url(null, 'tapestry-tool/v1'); ?>";
             var adminAjaxUrl = "<?php echo admin_url('admin-ajax.php'); ?>";
-
-            // Capture click events anywhere inside or outside tapestry
-            $(document).ready(function() {
-                document.body.addEventListener('click', function(event) {
-                    var x = event.clientX + $(window).scrollLeft();
-                    var y = event.clientY + $(window).scrollTop();
-                    recordAnalyticsEvent('user', 'click', 'screen', null, {
-                        'x': x,
-                        'y': y
-                    });
-                }, true);
-
-                document.getElementById('tapestry').addEventListener('click', function(event) {
-                    var x = event.clientX + $(window).scrollLeft();
-                    var y = event.clientY + $(window).scrollTop();
-                    recordAnalyticsEvent('user', 'click', 'tapestry', null, { x, y });
-                }, true);
-            });
```

Comment:
Yes -- It was throwing errors because `recordAnalyticsEvent` was defined inside `tapestry.js`. I moved it to `TapestryApp.vue` so all the code lives in one place.